cfgov/unprocessed/css/header.less

```diff
@@ -51,10 +51,11 @@
     width: 190px;
     height: 40px;
     margin: 10px;
+    margin-top: 0;
     vertical-align: middle; // removes typical inline vertical whitespace
 
     .respond-to-min(@bp-med-min, {
-        margin: @grid_gutter-width 0 unit(20px / @base-font-size-px, em) 0;
+        margin: 0 0 unit(20px / @base-font-size-px, em) 0;
```

Comment:
Let's come back to this. These header styles need to migrate over to `/organisms/header.less`, which will happen in another PR. This was updated here to remove the spacing that is added by the atomic header.